sdk/search/azure-search-documents/azure/search/documents/_service/_search_service_client.py

```diff
@@ -0,0 +1,82 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+from typing import TYPE_CHECKING
+
+from azure.core.pipeline.policies import HeadersPolicy
+from azure.core.tracing.decorator import distributed_trace
+from ._generated import SearchServiceClient as _SearchServiceClient
+from .._version import VERSION
+
+if TYPE_CHECKING:
+    # pylint:disable=unused-import,ungrouped-imports
+    from typing import Any, Union
+    from .. import SearchApiKeyCredential
+
+
+class SearchServiceClient(object):
+    """A client to interact with an existing Azure search service.
+
+    :param endpoint: The URL endpoint of an Azure search service
+    :type endpoint: str
+    :param credential: A credential to authorize search client requests
+    :type credential: SearchApiKeyCredential
+
+    .. admonition:: Example:
+
+        .. literalinclude:: ../samples/sample_authentication.py
+            :start-after: [START create_search_service_client_with_key]
+            :end-before: [END create_search_service_client_with_key]
+            :language: python
+            :dedent: 4
+            :caption: Creating the SearchServiceClient with an API key.
+    """
+
+    def __init__(self, endpoint, credential, **kwargs):
+        # type: (str, SearchApiKeyCredential, **Any) -> None
```

Comment:
@brjohnstmsft, are there any plans to support token credentials for search?

Reply:
https://github.com/Azure/azure-sdk-for-python/issues/10388

Reply:
@johanste I'm pretty sure the answer is yes, since we're looking at adding AAD authentication to the service this year.